pkg/features/features.go

```diff
@@ -44,3 +50,12 @@ func saveFeature(name, envVar string, defaultValue, overridable bool) FeatureFla
 	Flags[f.envVar] = f
 	return f
 }
+
+// LogFeatureFlags logs the global state of all features flags.
+func LogFeatureFlags() {
+	var context []interface{}
+	for _, flag := range Flags {
+		context = append(context, flag.LoggingContext())
+	}
+	log.Infow("Feature flags", context...)
```

Comment:
I think it should be multiline to be easier to read by human and spot configuration on the first glance. How about just `<flag name>: <enabled>` sorted with a flag per line?